src/go/rdctl/pkg/shutdown/shutdown.go

```diff
@@ -110,26 +122,34 @@ func checkProcessQemu() bool {
 	return checkProcessLinuxLike("-f", RancherDesktopQemuCommand)
 }
 
-func pkillQemu() {
+func pkillQemu() error {
 	cmd := exec.Command("pkill", "-9", "-f", RancherDesktopQemuCommand)
 	cmd.Stdout = os.Stdout
 	cmd.Stderr = os.Stderr
 	err := cmd.Run()
 	if err != nil {
-		logrus.Errorf("Failed to kill qemu: %s", err)
+		return fmt.Errorf("failed to kill qemu: %s", err)
```

Comment:
Preferable to use the `%w` verb for errors in `fmt.Errorf` calls

https://go.dev/blog/go1.13-errors#wrapping-errors-with-w